Snippets/Extensions.Hosting/Extensions.Hosting_1/Configuration.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Extensions.Hosting_1
+{
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+    using NServiceBus;
+
+    class Configuration
+    {
+        async Task ConfigureHost()
+        {
+            #region extensions-host-configuration
+            var host = new HostBuilder()
+                .ConfigureServices(serviceCollection =>
+                {
+                    var endpointConfiguration = new EndpointConfiguration("MyEndpoint");
+                    // configure endpoint here
+
+                    serviceCollection.AddNServiceBus(endpointConfiguration);
```

Comment:
I'm assuming we'll change this to `host.UseNServiceBus(...)` once https://github.com/Particular/NServiceBus.Extensions.Hosting/pull/13 is merged.

Reply:
Nevermind. This snippet will stay as-is for the WebHost. We'll add another one for the GenericHost.